tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
This one come to my mind after I read the @isimluk note. I think if would be better then the first solution. May @mpreisler or @ybznek could give us their opinions.